superset/assets/src/SqlLab/components/TabbedSqlEditors.jsx

```diff
@@ -145,6 +145,13 @@ class TabbedSqlEditors extends React.PureComponent {
   handleSelect(key) {
     if (key === 'add_tab') {
       this.newQueryEditor();
+    } else if (key === 'close_all_tabs') {
+      if (window.confirm('Are you sure you wish to clear all query editors?')) {
```

Comment:
Here it would be better to use the `<Dialog>` component, like you mentioned in Slack. You can see how it's used here: https://github.com/apache/incubator-superset/blob/8307716d723d7e16d3e5dd579a6d9e0f8e5413d8/superset/assets/src/datasource/DatasourceModal.jsx#L42